### PR TITLE
Update connection and connector API docs.

### PR DESCRIPTION
**Pull request by lmontopo**
Update docs to reflect Connectors endpoint being writeable, and having 'params'. 

Refs: #SDE-3461, #SDE-3462

### CODE REVIEW

**Comment by lmontopo** [comment]
@Gee19 I think I've finished updated the docs.  
Let me know if you think I'm missing anything.  If I'm not mistaken, I don't think the connections endpoint needs to be updated.

File: source/includes/connectors.md [hunk 1]
```diff
@@ -74,7 +107,95 @@ Returns a specific ALM connector.
 
 
 
+## Create an ALM Connectors
```
Comment (Gee19) [inline source/includes/connectors.md]: Connectors should be singular here

**Comment by Gee19** [comment]
Left one comment, and I think you are correct we do not have to update connections as it was always updatable(TIL how to spell this word).

+1 after you address my comment and fix the conflicts.

File: source/includes/connectors.md (hunk 1 above)
Comment (lmontopo) [inline source/includes/connectors.md]: agreed.  Thanks!

**Pull request merged**